docs/_partials/actions/configuration/related_models.rst

```diff
@@ -1,4 +1,13 @@
-Related models
-^^^^^^^^^^^^^^
+relatedModels
+^^^^^^^^^^^^^
 
-related_models
+.. note::
+
+    If you have the :doc:`RelatedModels listener </listeners/related-models>` configured, you can have Crud automatically load related data
```

Comment:
Yes but could be a : or something like "as shown below:" as well. 

I made a lot more comments mentioning dots, please read the as "full stop"